bin/e2e

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
-if [ "$KNAPSACK_PRO_TEST_SUITE_TOKEN_CYPRESS" = "" ]; then
-  KNAPSACK_PRO_ENDPOINT=https://api-disabled-for-fork.knapsackpro.com \
-    KNAPSACK_PRO_MAX_REQUEST_RETRIES=0 \
-    KNAPSACK_PRO_TEST_SUITE_TOKEN_CYPRESS=disabled-for-fork \
-    npx cypress run # run without parallelization
-else
-  npx knapsack-pro-cypress # parallelization is available
-fi
+RAILS_ENV=test bundle exec rake cypress:run # TODO: Fix parallelization
```

Comment:
I'm still reworking this as I need to get the cypress-rails rake task working with knapsack pro.